index.js

```diff
@@ -29,6 +29,19 @@ var instructions = [
 ];
 
 /**
+ * Extract protocol information from a URL
+ * Correctly extracts protocols with/without double slash ("//");
```

Comment:
It is meant to be an internal/private function not exposed in the API. I can still use JSDoc for it I guess